sale_commission_brand/models/account_invoice.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, models
+
+
+class AccountInvoiceLineAgent(models.Model):
+    _inherit = "account.invoice.line.agent"
+
+    @api.depends('invoice_line.price_subtotal')
+    def _compute_amount(self):
+        for line in self:
+            line.amount = 0.0
+            if (not line.invoice_line.product_id.commission_free and
+                line.commission) and \
+                    line.commission.commission_type == 'brand':
```

Comment:
@robyf70 in this way you break normal commission computation. You need to include this line within the "if commission_type == 'brand'"

Reply:
I mean line 12, "line.amount = 0.0"

Reply:
I've just committed a new patch, please review